src/libraries/System.Threading/tests/MutexTests.cs

```diff
@@ -236,6 +236,7 @@ public void Ctor_InvalidNames_Unix()
 
         [Theory]
         [MemberData(nameof(GetValidNames))]
+        [ActiveIssue("https://github.com/dotnet/runtime/issues/114951", TestRuntimes.CoreCLR)]
```

Comment:
We do want to keep running these tests on all other CoreCLR OSes. We want them to be skipped on Android only. 

You can use two different styles to combine the conditions:
- Like this: https://github.com/dotnet/runtime/blob/100ba615526926ca71e44c492d6a14314325a901/src/libraries/System.Threading.Thread/tests/ThreadTests.cs#L918
- Or like this: https://github.com/dotnet/runtime/blob/100ba615526926ca71e44c492d6a14314325a901/src/libraries/System.Net.Http/tests/FunctionalTests/TelemetryTest.cs#L25